src/structlog/tracebacks.py

```diff
@@ -240,6 +247,24 @@ def extract(
             is_cause=is_cause,
         )
 
+        if sys.version_info >= (3, 11):
```

Comment:
Should this support python <3.11 through https://github.com/agronholm/exceptiongroup ?

Reply:
rich doesn't do it so I did't want to bother, too. :)